Content/posts/building-3238-packages-for-apple-silicon.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-08-20 12:00
+title: Building 3,238 Swift Packages for Apple Silicon
+description: As part of the Swift Package Index build system, we have processed what must be the most extensive test of Apple Silicon compatibility outside of Apple. Here's what we found.
+---
+
+As part of the [Swift Package Index build system](https://blog.swiftpackageindex.com/posts/launching-language-and-platform-package-compatibility/), we have processed what must be the most extensive test of Apple Silicon compatibility outside of Apple.
+
+At the time of writing, we currently have XX,XXX Apple Silicon builds in our database covering 3,238 packages. Why so many? We test up to [three versions of each package](https://swiftpackageindex.com/SwifQL/SwifQL/builds) and compile each with both `xcodebuild` and `swift build`.
+
+I’m sure that many of the Apple Silicon DTK machines Apple has shipped around the world are working very hard, testing apps for compatibility, but I’m not sure any of them have worked quite as hard as ours has. 🚀
+
+## What did we find?
+
+First of all, and as you might expect, there’s plenty of good news. For packages with successful Intel builds, the *vast majority* are also successful when built for Apple Silicon. 👍
+
+There are failures though, and the most interesting are those where the build fails on Apple Silicon while the corresponding Intel build succeeds. In cases like this, that indicates a compatibility issue that will potentially need some attention from the package author. We’ve identified 70 packages (~2%) where this is the case, and have [compiled a list of them for reference](https://example.com/link/to/the/gist).
```

Comment:
Also update these numbers, if necessary.

Reply:
This line also includes the URL to the gist.

Reply:
I had a copy paste error in my query (both SPM and xcodebuild) where showing SPM builds. Will need to take another look at result!